tests/cli-exact.rs

```diff
@@ -117,10 +117,19 @@ r"",
 fn list_overrides() {
     setup(&|config| {
         let cwd = std::fs::canonicalize(env::current_dir().unwrap()).unwrap();
+        let cwd_fmt = format!("{}", cwd.display()).to_string();
+        
+        let unc_present = cwd_fmt.find(r"\\?\");
+    
+        let cwd_formatted = match unc_present {
+            None => cwd_fmt,
+            Some(_) => cwd_fmt[4..].to_owned(),
+        };
```

Comment:
Instead of adding this logic that mirrors the implementation, can you make it so if `cfg!(windows)` it unconditionally strips the 4 bytes? That assumes the test suite will never be run on the network, but that's fine.